src/component/Header/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC} from 'react';
+import './Header.scss';
+import './HeaderResponsive.scss';
+import {Logo} from "../Logo/Logo";
+import {UserProfile} from "../UserProfile/UserProfile";
+import { NavLink } from 'react-router-dom';
+
+export const Header: FC = () => {
+    return (
+        <>
+            <div id="main-header" className="main-header w-100 main-bg">
+                <div className="container">
+                    <div className="header-content-wrap">
+                        {/*End Component*/}
+                        <Logo/>
+                        {/*End Logo Component*/}
+                        {/*Start Header Navigation*/}
+                        <div className="header-navigation">

```

Comment:
Use HTML5 nav tag instead of div for app navigation.

Reply:
Nav tag has been added for header navigation